contracts/Level0Resources.sol

```diff
@@ -5,69 +5,69 @@ pragma solidity 0.8.0;
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "./OrderBook.sol";
 
-contract Zones is Ownable {
-    struct Zone {
+contract Level0Resources is Ownable {
+    struct Level0Resource {
         bytes32 identifier;
-        bool zoneExists;
+        bool level0ResourceExists;
         uint256 supply;
         uint256 min;
         uint256 max;
     }
```

Comment:
in solidity 8 this can be outside the contract

Reply:
Agree Lucas, we will take out all of them in another story.